examples/server-gke-vpc-native.yaml

```diff
@@ -0,0 +1,45 @@
+# Deploy LiveKit into a VPC-native GKE cluster
+# see https://cloud.google.com/kubernetes-engine/docs/concepts/alias-ips
+
+replicaCount: 2
+
+# Refer to https://docs.livekit.io/deploy/kubernetes/ for instructions
+
+livekit:
+  rtc:
+    use_external_ip: true
+  redis:
+    address: <redis-address>:6379
+  keys:
+    <key>: <secret>
+  turn:
+    enabled: true
+    domain: <turn-address>
+    tls_port: 3478
+    secretName: <turn-tls-secret>
+
+loadBalancer:
+  type: gke-vpc-native
+  tls:
+    - hosts:
+        - <server-address>
+      secretName: <server-tls-secret>
+
+service: 
+  annotations:
+    cloud.google.com/neg: '{"ingress": true}'
```

Comment:
Now that we have the type `gke-vpc-native` - should we add this annotation automatically?